src/signalfx_metadata.py

```diff
@@ -883,6 +890,39 @@ def plugin_config(conf):
     get_aws_info()
 
 
+def load_persistent_data():
+    """Load persistent data from a specified path"""
+    try:
+        with open(os.path.join(PERSISTENCE_PATH, PERSISTENCE_FILE), 'r') as js:
+            persist = json.load(js)
+            debug("Loaded the following persistent data %s" % persist)
+            if "NEXT_METADATA_SEND" in persist:
+                global NEXT_METADATA_SEND
+                NEXT_METADATA_SEND = persist["NEXT_METADATA_SEND"]
+            if "NEXT_METADATA_SEND_INTERVAL" in persist:
+                global NEXT_METADATA_SEND_INTERVAL
+                NEXT_METADATA_SEND_INTERVAL = \
+                    persist["NEXT_METADATA_SEND_INTERVAL"]
```

Comment:
@mdubbyap I have addressed your comment regarding host name.  Please see the latest commit to this pr.  Host name is now loaded and saved, and then checked when `steal_host_from_notifications()` is invoked.  If the loaded host name doesn't match, the metadata interval and next metadata send time are reset.  This should reset things prior to the first time the metadata code path is executed.